napari_allencell_annotator/_tests/view/annotator_view_test.py

```diff
@@ -12,6 +24,51 @@
 from napari_allencell_annotator.widgets.template_item import TemplateItem
 
 
+@pytest.fixture
+def annotator_model(qtbot) -> AnnotatorModel:
+    return AnnotatorModel()
+
+
+@pytest.fixture
+def viewer(qtbot) -> IViewer:
+    return FakeViewer()
+
+
+@pytest.fixture
+def annotator_view(qtbot, annotator_model: AnnotatorModel, viewer: IViewer) -> AnnotatorView:
+    return AnnotatorView(annotator_model, viewer)
+
+
+def test_render_values(annotator_view: AnnotatorView) -> None:
+    # ARRANGE
+    annotator_view.annot_list.add_item("text", Key("string", "text"))
+    annotator_view.annot_list.add_item("number", Key("number", 1))
+    annotator_view.annot_list.add_item("point_created", Key("point", None))
```

Comment:
if you change the fixture in a test, there's no point in creating that fixture in the first place. Just initialize annotator view in the test and arrange it for the test.

Reply:
changed in https://github.com/aics-int/napari-allencell-annotator/pull/94/commits/eda619feae1f82dcd7f8bfde3f2c7730a548d432